src/javaRosa.js

```diff
@@ -1613,6 +1694,13 @@ define([
                             xmlWriter.writeXML(xml.normalize(val));
                             xmlWriter.writeEndElement();
                         }
+                        if (writeMarkdown) {
+                            val = forms[0].getValueOrDefault(lang);
```

Comment:
`forms[0].getValueOrDefault(lang)` doesn't look right because you don't know which form `form[0]` is, but I could be mistaken. Are you looking for the default form? In that case you probably want `itext.get('default', lang)`, right?